megamek/src/megamek/client/ui/swing/MiniMap.java

```diff
@@ -1344,316 +977,261 @@ private void paintUnit(Graphics g, Entity entity) {
                     s = STRAT_WEIGHTS[entity.getWeightClass()];
                 }
                 if (!s.equals("")) {
-                    FontRenderContext fontContext = new FontRenderContext(null, true, true);
-                    Font font = new Font("SansSerif", Font.BOLD, 100);
-                    FontMetrics currentMetrics = getFontMetrics(font);
-                    int stringWidth = currentMetrics.stringWidth(s);
+                    var fontContext = new FontRenderContext(null, true, true);
+                    var font = new Font("SansSerif", Font.BOLD, 100);
+                    var currentMetrics = getFontMetrics(font);
+                    var stringWidth = currentMetrics.stringWidth(s);
```

Comment:
Is `var` supported in Java 11 outside of lambda parameters?

Reply:
If it is, my preference for `var` is only when the type is obvious (i.e. the typename itself is on the RHS):
```suggestion
                    var fontContext = new FontRenderContext(null, true, true);
                    var font = new Font("SansSerif", Font.BOLD, 100);
                    FontMetrics currentMetrics = getFontMetrics(font);
                    int stringWidth = currentMetrics.stringWidth(s);
```